openapi/components/schemas/RuleActionAdjustReadyToPayout.yaml

```
@@ -0,0 +1,14 @@
+allOf:
+  - $ref: ./RuleAction.yaml
+  - type: object
+    description: Adjust ready to payout.
+    properties:
+      prioritizeActivePaymentInstruments:
+        type: boolean
+        default: false
+        description: If set as `true`, the payment methods associated to the customer's active payment instruments are displayed at the top of the list.
+      paymentMethods:
+        type: array
+        description: Ordered list of payment methods allowed. If empty, then no payment methods are allowed.
```

suggestion: for readability
```suggestion
        description: Ordered list of allowed payment methods. If this field is empty, no payment methods are allowed.
```